src/card-manager.ts

```diff
@@ -1447,7 +1447,11 @@ export class CardManager implements Initializable, NotificationHandler, CardHand
         }
         const userProperties = await db.findCardProperties(card.id, "user", user.id);
         for (const property of userProperties) {
-          card.state.user.properties[property.name] = property.value;
+          let value = property.value;
+          if (typeof value === 'string') {
+            value = fileManager.rewriteFileUrls(value as string);
+          }
+          card.state.user.properties[property.name] = value;
         }
```

Comment:
ALL user string properties cannot be URLs (in the future)

Reply:
I see you are returning the same string when not URL. Never mind then. Just hard to read. 
Maybe in the long term do this automatically on DB insertion?

Reply:
Rest looks fine